python/lsst/ip/isr/isrTaskLSST.py

```diff
@@ -1234,13 +1244,6 @@ def overscanCorrection(self, mode, detectorConfig, detector, badAmpDict, ccdExpo
 
         return overscans
 
-    def correctGains(self, exposure, ptc, gains):
-        # TODO DM 36639
```

Comment:
Please comment on DM-36639 that this has been removed.  Just so it's clear: will this new calibration type handle any future temperature correction as well, or are we dropping that entirely?

Reply:
The idea is that we can add the temperature support to this calibration type in the future if and when we do that.  I'll comment on the ticket.